tls/test/test_hello_message.py

```diff
@@ -35,6 +35,13 @@ class TestClientHello(object):
         b''  # extensions.extensions
     )
 
+    supported_signature_list_extension_data = (
+        b'\x00\x0D'  # extensions[0].extension_type
+        b'\x00\x16'  # extensions[0].length
+        b'\x00\x14\x04\x01\x05\x01\x06\x01\x02\x01\x04\x03\x05\x03\x06\x03'
```

Comment:
Good idea, thanks!

Could you share how you translated the bytes? I thought `b'\x02\x01'` should be `SHA1, RSA`?

Here's what I translated them into:
```    
supported_signature_list_extension_data = (
        b'\x00\x0D'  # extensions[0].extension_type
        b'\x00\x16'  # extensions[0].length
        b'\x00\x14'  # The length of signature_algorithms vector
        b'\x04\x01'  # SHA256, RSA
        b'\x05\x01'  # SHA384, RSA
        b'\x06\x01'  # SHA512, RSA
        b'\x02\x01'  # SHA1, RSA
        b'\x04\x03'  # SHA256, ECDSA
        b'\x05\x03'  # SHA384, ECDSA
        b'\x06\x03'  # SHA512, ECDSA
        b'\x02\x03'  # SHA1, ECDSA
        b'\x04\x02'  # SHA256, DSA
        b'\x02\x02'  # SHA1, DSA
    )
```
based on:

```
class HashAlgorithm(Enum):
    NONE = 0
    MD5 = 1
    SHA1 = 2
    SHA224 = 3
    SHA256 = 4
    SHA384 = 5
    SHA512 = 6


class SignatureAlgorithm(Enum):
    ANONYMOUS = 0
    RSA = 1
    DSA = 2
    ECDSA = 3
```